R/util.R

```diff
@@ -291,12 +291,12 @@ duration_to_minutes <- function(period, name = "testing") {
 
 
 format_datetime <- function(year, month, day, hour, minute, second) {
-  format(to_POSIXct(
+  format(to_posix_ct(
     sprintf("%s-%s-%s %s:%s:%s", year, month, day, hour, minute, second)),
     "%Y-%m-%d %H:%M:%S")
 }
 
-to_POSIXct <- function(s) {
```

Comment:
Hmm, not sure of these two since POSIXct is the name of the R type... but ok...!